zkevm-circuits/src/evm_circuit/util/common_gadget.rs

```diff
@@ -841,6 +841,12 @@ impl<F: Field> TransferToGadget<F> {
         } else {
             (0.into(), 0.into())
         };
+
+        #[cfg(feature = "l1_fee_curie")]
+        println!("l1_fee_curie enabled");
+        #[cfg(not(feature = "l1_fee_curie"))]
+        println!("l1_fee_curie not enabled");
```

Comment:
remove to switch to log::debug or remove finally

Reply:
removed